api/trace/sampler.go

```diff
@@ -26,6 +26,9 @@ type Sampler interface {
 		traceID core.TraceID,
 		spanID uint64,
```

Comment:
This uses `core.SpanID` elsewhere. Would you mind fixing this up?

Reply:
Done.